File: mobileBE/src/main/java/com/uth/mobileBE/dto/response/BookResponse.java
package com.uth.mobileBE.dto.response;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder
public class BookResponse {
    private Long bookId;
    private Long libraryId;
    private Long categoryId;
    private String isbn;
    private String title;
    private String author;
    private Double basePrice;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: app/src/main/java/com/example/quanlythuvien/data/model/response/BookResponse.kt
package com.example.quanlythuvien.data.model.response

data class BookResponse(
    val bookId: Long,
    val libraryId: Long? = null,
    val isbn: String,
    val title: String,
    val author: String,
    val basePrice: Double? = null,
    val availableCopies: Int? = null,
    val categoryId: Long? = null,
    val categoryName: String? = null
)